#Header header

string best_pick
int64 probability
duration scan_time

